src/main/groovy/com/palantir/gradle/gitversion/GitDescribe.groovy

```diff
@@ -0,0 +1,6 @@
+package com.palantir.gradle.gitversion
+
+interface GitDescribe {
+
+    String describe(String prefix)
```

Comment:
I think it would be helpful to add some javadoc explaining that this is effectively `git describe --first-parent --tags --always` where the tag matches the prefix